L/laz_perf/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "laz_perf"
```

Comment:
Maybe call it
```suggestion
name = "LAZperf"
```
as it's styled in the readme?  You'll need to rename the directory accordingly, in case

Reply:
I think I tried `laz-perf` originally but the wizard did not seem to like that? LAZperf also works fine.

```
			# Step 2b: Obtain binary dependencies (if any)

Do you require any (binary) dependencies?  [y/N]: n

[ Info: Updating bare Yggdrasil clone in deps/Yggdrasil...
Enter a name for this project.  This will be used for filenames:
> laz-perf

"laz-perf" is an invalid identifier. Try again.
Enter a name for this project.  This will be used for filenames:
```

I'm not sure why there is an extra commit in there, I tried to do a `git add * -u` and somehow added a merge commit.... Hopefully that doesn't hurt anything.

Reply:
We can't have a package called `laz-perf_jll`, that isn't a valid identifier:
```julia
julia> Meta.@dump using laz-perf_jll
Expr
  head: Symbol call
  args: Array{Any}((3,))
    1: Symbol -
    2: Expr
      head: Symbol using
      args: Array{Any}((1,))
        1: Expr
          head: Symbol .
          args: Array{Any}((1,))
            1: Symbol laz
    3: Symbol perf_jll
```